aiida/orm/nodes/data/array/kpoints.py

```diff
@@ -268,8 +262,7 @@ def set_kpoints_mesh(self, mesh, offset=[0., 0., 0.]):
         # I cannot have both of them at the same time
         try:
             _ = self.get_array('kpoints')
-            raise ModificationNotAllowed("KpointsData has already a kpoint-"
-                                         "list stored")
+            raise ModificationNotAllowed("KpointsData has already a kpoint-" "list stored")
```

Comment:
There is quite a few occurrences of `" "` due to `yapf`

Reply:
Good catch, will fix it